documentation/advanced-tooling-generic.asciidoc

```diff
@@ -13,6 +14,12 @@ To draw diagrams for your project or for blueprints in devonfw, we recommend the
 * https://www.objectaid.com/[ObjectAid] is a nice and easy to use eclipse plugin that you can use to quickly create UML diagrams from existing code. While class-diagrams are supported for free, you need to buy a license if you want to use the other diagram types.
 * https://plantuml.com/[PlantUML] is a great tool that can render UML diagrams from simple markup that can be easily managed in git or other version-control systems together with your code. Its simplicity allows branching and merging unlike other greedy binary UML data-formats.
 
-== Browser Plugins
-There are tons of helpful browser plugins out there and it might be a matter of personal taste what you like to have installed. However, as we are heavily using github we want to promote https://github.com/buunguyen/octotree#octotree[octotree].
-In case you also work with ZenHub you might want to install the https://www.zenhub.com/extension[Zenhub Browser Extension].
+== Git Client
+If you are looking for a git client that works cross-platform we recommend to use https://git-fork.com/[Fork].
+
+
+== Tab Grouping For The Web Browser
+https://addons.mozilla.org/en-US/firefox/addon/simple-tab-groups/[Simple Tab Groups] is a Firefox plugin that helps to separate tabs e.g. tabs for projekt A, projekt B, travelplanning or hobbies 
+
+== Windows Terminal
+https://github.com/microsoft/terminal[Windows Terminal] is a tool that enables the user to use CLIs of various kinds like git-bash, Powershell, CMD, WSL-Shells or Putty
```

Comment:
This is still missplaced here. Please remove.
Feel free to improve here:
https://github.com/devonfw/ide/blob/master/documentation/advanced-tooling-windows.asciidoc#tabs-for-cmd

Reply:
Thanks for the improvement. Done 👍